.github/workflows/test-publish.yml

```diff
@@ -22,15 +22,15 @@ jobs:
       - name: Get current month to build the cache key
         run: echo "currentMonth=$(date +'%Y-%m')" >> $GITHUB_ENV
         shell: bash
-      - name: Cache test-app yarn
+      - name: Cache it-tests yarn
         uses: actions/cache@v3
         with:
           path: |
-            .cache/test-app
-            !.cache/test-app/@ama-sdk*
-            !.cache/test-app/@ama-terasu*
-            !.cache/test-app/@o3r*
-          key: ${{ runner.os }}-test-app-yarn3-${{ env.currentMonth }}
+            .cache/test-*
+            !.cache/test-*/@ama-sdk*
+            !.cache/test-*/@ama-terasu*
+            !.cache/test-*/@o3r*
+          key: ${{ runner.os }}-it-tests-yarn3-${{ env.currentMonth }}
```

Comment:
there is a chance that one of the it-tests will not be executed (due to nx cache) when registering the cache, in which case the cache will be missing for this test the whole month
maybe we should separate the cache for test-app and test-sdk ?